src/main/java/edu/umich/ctools/sectionsUtilityTool/Utils.java

```diff
@@ -22,4 +31,129 @@ public static Properties loadProperties(String path){
 		M_log.error("File path for (" + path + ") is not provided");
 		return null;		
 	}
-}
+
+	public static void openFile(HttpServletRequest request,
+			HttpServletResponse response, PrintWriter out) {
+		M_log.debug("Stub method called");
+		String pathInfo = request.getPathInfo();
+		M_log.debug("Path Info: " + pathInfo);
+		
+		String url;
+		String queryString = request.getQueryString();
+		
+		if(queryString!=null) {
+			url=pathInfo+"?"+queryString;
+		}else {
+			url=pathInfo;
+		}
+		
+		File testFile = null;
+		FileReader fr = null;
+
+		if (url.matches(SectionsUtilityToolServlet.appExtPropertiesFile.getProperty(CANVAS_API_TERMS))){
```

Comment:
Way too much repeated code.  A couple ways of dealing with that are:
- refactor the try / catch into a method
- Put the pathInfo and associated resource file name into an array and just loop through the array, processing when there is a match.